app/src/main/java/org/wikipedia/analytics/eventplatform/WatchlistAnalyticsHelper.kt

```diff
@@ -0,0 +1,51 @@
+package org.wikipedia.analytics.eventplatform
+
+import android.content.Context
+import org.wikipedia.WikipediaApp
+
+object WatchlistAnalyticsHelper {
+    fun logAddedToWatchlist(context: Context, itemWiki: String) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "Watchlist.itemWiki:$itemWiki.added"
+            )
+        )
+    }
+
+    fun logRemovedFromWatchlist(context: Context, itemWiki: String) {
```

Comment:
Instead of an `itemWiki: String` parameter for all of these functions, a slightly cleaner way is to take a `title: PageTitle?` parameter. The caller doesn't need to know which parts of `PageTitle` will be used here.